### PR TITLE
Added Restart Recovery section

### PR DESCRIPTION
**Pull request by ramonsmits**
Guidance on how to set windows service recovery options for windows services using sc.exe

### CODE REVIEW

**Comment by SimonCropp** [comment]
@ramonsmits  can u fix the build

**Comment by SimonCropp** [comment]
add a link here https://technet.microsoft.com/en-us/library/cc742019.aspx

**Comment by gbiellem** [comment]
@ramonsmits  :+1:  

Be nice if this included how to do it via the gui service manager as well

File: nservicebus/hosting/windows-service.md [hunk 1]
```diff
@@ -107,6 +107,17 @@ sc.exe config [ServiceName] depend= <Dependencies(separated by / (forward slash)
 sc.exe config SalesEndpoint depend= MSMQ/MSDTC/RavenDB
 ```
 
+### Restart Recovery
+
+The endpoint instance may decide to quit due to its internal circuit breaker behavior. When an NServiceBus endpoint is hosted as a Windows Service is it important to configure the Windows Service Recovery options. These can be set by the service properties or via `sc.exe` which has advanced configuration options.
```
Comment (ramonsmits) [inline nservicebus/hosting/windows-service.md]: When using the host it is the default behavior. When self hosting the instance becomes a zombie. I'll try and improve this.

Comment (SimonCropp) [inline nservicebus/hosting/windows-service.md]: This page is explicitly talking about self hosting.

File: nservicebus/hosting/windows-service.md [hunk 2]
```diff
@@ -112,9 +112,11 @@ sc.exe config SalesEndpoint depend= MSMQ/MSDTC/RavenDB
 
 Windows has a Windows Service Recovery mechanism that makes sure that a crashed service will be restarted.
 
-The endpoint can fail when using the [NServiceBus Host](nservicebus-host/) or when [self hosting and implementing a critical error handler that exits the process](critical-errors.md#default-action-handling-in-nservicebus) in case a critical error occurs. Not having Windows Service Recovery configured will result in message processing to halt.
```
Comment (ramonsmits) [inline nservicebus/hosting/windows-service.md]: @weralabaj Why was the last sentence removed?

Comment (weralabaj) [inline nservicebus/hosting/windows-service.md]: It wasn't removed, it was rephrased and moved to the next section ;)

Comment (ramonsmits) [inline nservicebus/hosting/windows-service.md]: @weralabaj Didn't notice that sorry :(

**Pull request merged**